sqlglot/dialects/presto.py

```diff
@@ -198,6 +198,7 @@ class Presto(Dialect):
     TYPED_DIVISION = True
     TABLESAMPLE_SIZE_IS_PERCENT = True
     LOG_BASE_FIRST: t.Optional[bool] = None
+    SUPPORTS_VALUES_DEFAULT = False
```

Comment:
Both Presto and Trino treat `DEFAULT` as an identifier: I tried running the issue's `INSERT` and got an identifier resolution error back. Only those two out of the engines I tested had this behavior, the other engines treat `DEFAULT` as a special keyword in the context of a `VALUES` clause.